src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -68,7 +68,16 @@
       (compojure/GET "/services" []
         (let [statuses (call-status-fns status-fns-atom)]
           {:status 200
-           :body {"services" statuses}})))))
+           :body statuses}))
+       (compojure/GET "/services/:service-name" [service-name]
+         (if-let [status-info (get (deref status-fns-atom) service-name)]
```

Comment:
I think a better name would be `service-info`, the corresponding schema is `ServiceInfo`.